Assets/Scripts/UI/ObjectiveList/ObjectiveList.cs

```diff
@@ -10,6 +10,10 @@ public class ObjectiveList : Singleton<ObjectiveList>
     private TMP_Text objectiveText;
     private Animator root;
 
+    private float previousAxis;
+    private float currentAxis;
+    private const float LEFT_DPAD = 0.75;

```

Comment:
was this supposed to be  -0.75?

Reply:
also we can keep this value if you just rename it to like `DPAD_PRESS_THRESHOLD` or something! and then also apply the over changes below!